.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+This channel is used to report issues. Please, make sure that we do not have 
+any duplicates already open. You can ensure this by searching the issue list 
+for this repository. If there is a duplicate, please close your issue and add 
+a comment to the existing issue instead.
+
+If your report is not an issue but a question or doubt regarding Licode 
+documentation, features, roadmap, etc. please, use our Discourse channel in: 
+http://discourse.lynckia.com/
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+**Steps to reproduce the issue:**
+1.
+2.
+3.
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Additional information you deem important (e.g. issue happens only occasionally):**
+
+
+**Licode commit/release where the issue is happening**
+
+
+**Additional environment details (AWS, VirtualBox, physical, etc.):**
```

Comment:
Consider changing "physical" for "docker" here. I know it's probably obvious but I like it better